app/src/main/java/com/bottlerocketstudios/brarchitecture/ui/snippet/SnippetsViewModel.kt

```diff
@@ -19,13 +20,17 @@ class SnippetsViewModel : BaseViewModel() {
     val snippets: Flow<List<SnippetUiModel>> = repo.snippets.map { it ->
         it.map {
             SnippetUiModel(
+                id = it.id ?: "",
+                workspaceSlug = it.workspace?.slug ?: "",
                 title = it.title ?: "",
                 userName = it.owner?.displayName ?: "",
                 formattedLastUpdatedTime = it.updated.formattedUpdateTime(clock = clock)
             )
         }
     }
 
+    val showCreateCTA = MutableStateFlow(true)
```

Comment:
@BottleRocket-Colin nit: Refactor to `showCreateCta` (per guidance at https://developer.android.com/kotlin/style-guide#camel_case)

Reply:
Will do